apps/src/StudioApp.js

```diff
@@ -1456,7 +1456,7 @@ StudioApp.prototype.configureDom = function (config) {
     visualizationColumn.className = visualizationColumn.className + " embed_hidesource";
   }
 
-  if (!config.embed && !config.hideSource) {
+  if (!config.share) {
```

Comment:
I think that without this we weren't getting the correct sizing of the playspace (it had a gap at the bottom for a non-existent small footer). The goal here is that this look just like the normal /edit page, but without the source, so keeping the same styles seems appropriate.